### PR TITLE
lua53: fix "default-names" option for luarocks

### PR DESCRIPTION
**Pull request by telemachus**
Previously luarocks was installed as luarocks-5.3 and luarocks-admin-5.3
even if the user passed the `--with-default-names` switch.

### CODE REVIEW

File: lua53.rb [hunk 1]
```diff
@@ -79,8 +79,13 @@ def install
...
         (share/"lua/5.3/luarocks").install_symlink Dir["#{libexec}/share/lua/5.3/luarocks/*"]
-        bin.install_symlink libexec/"bin/luarocks-5.3"
-        bin.install_symlink libexec/"bin/luarocks-admin-5.3"
+        if build.with? "default-names"
+          bin.install_symlink libexec/"bin/luarocks"
+          bin.install_symlink libexec/"bin/luarocks-admin"
+        else
+          bin.install_symlink libexec/"bin/luarocks-5.3"
+          bin.install_symlink libexec/"bin/luarocks-admin-5.3"
+        end
```
Comment (ilovezfs) [inline lua53.rb]:
```
suffix = "-5.3" if build.without? "default-names"
bin.install_symlink libexec/"bin/luarocks#{suffix}"
bin.install_symlink libexec/"bin/luarocks-admin#{suffix}"
```

Comment (telemachus) [inline lua53.rb]: Yup: that's neater. I did it that way and then amended the commit. (I had trouble when trying string interpolation earlier, but I must have done something else wrong. Thanks.)

Comment (ilovezfs) [inline lua53.rb]: no problem ... thank you for opening this!

**Comment by MikeMcQuaid** [comment]
Thanks again @telemachus!

**Pull request merged**